docs/source/release-notes.rst

```diff
@@ -8,6 +8,7 @@ See what's been released in Vircadia.
     :maxdepth: 2
     :titlesonly:
 
+    Release 2021.1.0 Eos (2021-02-12) <release-notes/2021-1-0-Eos>
```

Comment:
I can understand moving away from "Alpha", but I feel like we are in "Beta" rather than a full release.
```suggestion
    Beta Release 2021.1.0 Eos (2021-02-12) <release-notes/2021-1-0-Eos>
```

Reply:
At a developer or community meeting not so long ago, a general consensus was reached to to remove such labels - the software is what it is. And it has been released. It's certainly usable; fine for some use cases, room for improvement in other use cases.

Reply:
Yeppers, basically the idea is that we have to be **very** clear on what parts of Vircadia are capable/ready for what use cases.

This page is meant to help indicate this https://vircadia.com/what-is-vircadia/ by outlining what it is and what it isn't respectively. I'll keep at updating it regularly especially given feedback on the various parts of the system.